flutter_common/lib/data/result_filter.dart

```diff
@@ -16,6 +17,7 @@ class ResultFilter {
   String? manufacturer;
   String? soc;
   String? benchmarkId;
+  SortByValues? sortBy;
```

Comment:
The sort should not be dependent on the filter or part of it.

Reply:
Refactored